app/controllers/registrations_controller.rb

```diff
@@ -4,10 +4,6 @@ class RegistrationsController < Devise::RegistrationsController
    # To redirct users after sign-up to edit
 
   def after_sign_up_path_for(user)
-    if current_user.freelancer
-      edit_freelancer_path(user.freelancer.id)
-    else
-      edit_client_path(user.client.id)
-    end
+    edit_freelancer_path(user.freelancer.id)
```

Comment:
nope, the invitable is not actually a sign-up, but an after accept invitation. So only the people that sign-up, are freelancers and can edit their freelancers profile